src/main/java/com/Studentlifr/GratitudeBoard/services/UserService.java

```diff
@@ -0,0 +1,19 @@
+package com.Studentlifr.GratitudeBoard.services;
+
+import com.Studentlifr.GratitudeBoard.dao.UserDAO;
+import com.Studentlifr.GratitudeBoard.dto.requests.UserSignupReqDTO;
+import com.Studentlifr.GratitudeBoard.dto.response.GenericResDTO;
+import com.Studentlifr.GratitudeBoard.dto.response.ResponseDTO;
+import com.Studentlifr.GratitudeBoard.services.impl.UserServiceImpl;
+
+import java.util.UUID;
+
+public interface UserService {
+    ResponseDTO signup(UserSignupReqDTO userSignupReqDTO);
+
+    UserDAO getUserByEmailId(String emailId);
+    ResponseDTO MakeVerified(String mailId);
+
+    UserDAO getUserById(UUID uid);
+    public Boolean isUserVerified(String emailId);
```

Comment:
remove public here as any method declared here is public by default

Reply:
Removing Public from there, restricting me to access the user service package in controller, (Outside the Package).